typings/objection/index.d.ts

```diff
@@ -14,10 +14,17 @@ declare namespace Objection {
   const compose: Compose;
   const mixin: Mixin;
 
+  const snakeCaseMappers: () => ColumnNameMappers;
+  const knexSnakeCaseMappers: () => KnexMappers;
+
   interface LiteralObject {
     [key: string]: Value;
   }
 
+  export interface AnyObject {
+    [key: string]: any;
```

Comment:
Should the value here really be any? Like, is it OK to have an array of model objects (not just object representations of model objects, but actual instances of Model subclasses)? It should nest other instances, correct?

See https://github.com/Vincit/objection.js/issues/558#issue-265887726 for a more constrained "AnyObject" definition, if we need that.

I think we can do better with the name, too, given that "object" already means two things in [TypeScript](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-2.html#object-type) (with `object` and `Object`) and `any` is a superset of those, so it's compounding two names with three definitions. 

Is `SimpleObject` better?

```ts
  type SimpleValue = string | number | boolean | Date | null | undefined | SimpleObject 

  interface SimpleObject {
    [key: string]: SimpleValue | SimpleValue[]
  }
```

Reply:
Usually the objects are JSON objects, but they can be basically anything. Database drivers can have their own conversions and users can pass anything to `fromJson` etc. Also I don't really see the use of limiting that type. Can you give me an example where that would be useful?

Reply:
You are right about the name, but I don't think SimpleObject is better. The object may be very complex.

Reply:
What about `Pojo`?

Reply:
or `PlainObject`?

Reply:
"Pojo" is perfect.